projects/packages/my-jetpack/package.json

```diff
@@ -43,6 +43,7 @@
 		"react-dom": "17.0.2",
 		"sass": "1.43.3",
 		"sass-loader": "12.4.0",
+		"storybook-addon-mock": "2.0.2",
```

Comment:
I think this dep should be added into `storybook` dependencies, this way it will be accessible for all projects.

Reply:
probably yes Wondering whether it could be done in a follow-up PR

Reply:
Tracking here for future PR